Scrabble/tam670.py

```diff
@@ -1,66 +1,141 @@
 import sys, math
 # # Max Scrabble Score Calculator (scrabble.py)
+#
 # ## Problem and Approach
-# This challenge asked us to find the word with the highest possible scrabble score from the set of 7 tiles given to us. My approach is to compare each word in the given dictionary to my tiles, letter by letter. This will ensure that I can find all the valid words that I can make with letters. Then we can calucalte the scores based off the below table. 
-
-# | Points | Letters |
-# | --- | --- | 
-# | 10  | q, z  |
-# | 8 | j, x| 
-# | 5 | k |
-# | 4 | f, h, v, w, y |
-# | 3 | b, c, m, p |
-# | 2 | d, g |
-# | 1 | a, e, i, o, u, l, n, s, t, r |
+#
+# This challenge asked us to find the valid word in the given dictionary with
+# the highest possible scrabble score from the set of 7 tiles given to us. The
+# dictionary is a list of strings given to us by the game. The score is
+# calculated by tallying up the point values of each letter in the word (see
+# corresponding chart below).  My approach is to compare each word in the given
+# dictionary to my tiles, letter by letter. This will ensure that I can find all
+# <span style="text-decoration: underline;">of</span> the valid words that I can
```

Comment:
Underline is messy in Markdown. Choose something else (_this_, for example, which is `_this_` in Markdown.).